### PR TITLE
Feature/36: ImportType logic

### PR DESCRIPTION
**Pull request by filippomc**
The logic for the import type message logic has been implemented, as well as the auto loading at startup with autoresolve logic

### CODE REVIEW

File: pygeppetto/managers/runtime_project.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+        # Set the new value in replacement of ImportValue
+        variable.initialValues[0].value = new_value
+        # TODO? it would be nice here to have an api that sets a value and also handles the sync
+        variable_container.synched = False
+        source_library.synched = False
+        return self.model
+
+    def resolve_import_type(self, typePaths):
+        """Loads a new Geppetto Model"""
```
Comment (tarelli) [inline pygeppetto/managers/runtime_project.py]: @filippomc wrong comment

File: pygeppetto/managers/runtime_project.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+        # TODO handle views
+        # TODO handle experiments
+
+    def release(self):
+        pass
+
+    def import_types(self, model):
+        model_traversal.apply(model, self.import_type,
+                              lambda eobject: type(eobject) == ImportType and eobject.autoresolve)
```
Comment (tarelli) [inline pygeppetto/managers/runtime_project.py]: rename eobject -> node?

File: pygeppetto/managers/runtime_project.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        self.geppetto_model_access = GeppettoModelAccess(self.model)
+        self.import_types(self.model)
+
+        # TODO handle views
+        # TODO handle experiments
+
+    def release(self):
+        pass
+
+    def import_types(self, model):
```
Comment (tarelli) [inline pygeppetto/managers/runtime_project.py]: rename to import_autoresolve_types?

File: tests/test_importvalue.py [hunk 4]
```diff
@@ -2,13 +2,16 @@
 from pygeppetto.constants import UserPrivileges
```
Comment (tarelli) [inline tests/test_importvalue.py]: @filippomc shall we add a test for importtypes?

**Review by tarelli — CHANGES_REQUESTED** [review]
Great work Filippo! Some minor comments.

File: tests/test_importvalue.py (hunk 4 above)
Comment (filippomc) [reply tests/test_importvalue.py]: Good point, I'll do it

Comment (filippomc) [reply tests/test_importvalue.py]: @tarelli test added, it was really useful to spot a problem that has been then solved.

**Pull request merged**